CHANGELOG.md

```diff
@@ -19,23 +23,25 @@ We've made the following adjustments to our responsive type scale:
 - point 36 remains as 27px on large screens
 - point 36 now returns 27px on small screens instead of 24px and has a line height 30px instead of 25px
 
-To enable these changes, set `$govuk-new-typography-scale` to `true` before you import GOV.UK Frontend in your Sass:
+To enable these changes, set the feature flag variable `$govuk-new-typography-scale` to true before you import GOV.UK Frontend in your Sass files:
```

Comment:
Not sure if this was an intentional change – think we would normally use inline code for `true` here?

```suggestion
To enable these changes, set the feature flag variable `$govuk-new-typography-scale` to `true` before you import GOV.UK Frontend in your Sass files:
```